tests/notebooks/training/requirements.in

```diff
@@ -1,2 +1,3 @@
-kubeflow-training
+# Pinning to 1.6.0 because of the breaking change with the imports in 1.7.0
+kubeflow-training==1.6.0
```

Comment:
Should we link https://github.com/canonical/charmed-kubeflow-uats/issues/84?

Reply:
Added Link